DevEdu.API/DevEdu.API/Models/OutputModels/Course/CourseInfoWithMaterialsAndTasksOutputModel.cs

```diff
@@ -2,7 +2,7 @@
 
 namespace DevEdu.API.Models.OutputModels
 {
-    public class CourseInfoWithMaterialsAndTasksOutputModel : CourseInfoOutputModel
+    public class CourseInfoWithMaterialsAndTasksOutputModel : CourseSimpleInfoOutputModel
```

Comment:
do we really need this model?

Reply:
It is not my, I,ve just got it from the main

Reply:
please delete this model as it is not used